views/dev_train_almond.pug

```diff
@@ -22,11 +22,11 @@ block content
       div.col-xs-12.col-md-8.col-lg-9
         section.divider#sentence-to-code-block
           h3= _("Train Almond")
-          p
-            = _("If Almond is misbehaving or misinterpreting your input, you can correct it here.")
+          p= _("If Almond is misbehaving or misinterpreting your input, you can correct it here.")
             |
             |
-            != _("Type a sentence and then press <kbd>Enter</kbd>. Then click on the corresponding interpretation, or use the ThingTalk options to write the corresponding code manually.")
+          p!= _("Try entering a verbal command by clicking the button below or pressing <kbd>g</kbd> to start and stop the recording! Then, just correct the transcription in the input box and press <kbd>Enter</kbd>. Alternatively, type a sentence and then press <kbd>Enter</kbd>.")
```

Comment:
To be honest I'm not sure using `g` to trigger the interaction is the best way. For one, it does not work on mobile. Also, it `g` as a letter is not a mnemonic. How about a button with a mic icon?

Reply:
Removed the key trigger